yb-voyager/src/srcdb/data/gather-assessment-metadata/oracle/yb-voyager-oracle-gather-assessment-metadata.sh

```diff
@@ -148,7 +155,8 @@ run_command() {
     # print and log the stderr/stdout of the command
     eval $command 2>&1 | tee -a "$LOG_FILE"
     if [ ${PIPESTATUS[0]} -ne 0 ]; then
-        print_and_log "ERROR" "command failed: $command"
+        print_and_log "ERROR" "command failed: $(remove_password "$command")"
+        printf "\n$(basename $file_to_print)\n" | tee -a "$LOG_FILE"
```

Comment:
what is this printf?

Reply:
it is printing the .csv file(containing error message) generated in case of sqlplus command, on to the console and log file. 
Will add a comment here.

Reply:
Got it. why not use print_and_log here as well? I assume with tee, that's what you're aiming to do: print to stdout and also to log file?

Reply:
print_and_log is using echo.
But i wanted to format the output, for eg there is a new line after file name, so used `printf`